pkg/occlient/occlient_test.go

```diff
@@ -1427,7 +1427,7 @@ func TestUpdateBuildConfig(t *testing.T) {
 					t.Errorf("deployment Config Spec not matching with expected values, expected: %s, got %s", tt.updatedBuildConfig.Spec, updatedDc.Spec)
 				}
 			} else if err == nil && tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "error", "no error")
+				t.Error("test failed, expected: error, got no error")
```

Comment:
When you see just this error it is not clear what happened.
"error was expected, but no error returned" or something like this